posthog/client.py

```diff
@@ -1003,10 +1300,7 @@ def get_feature_flag_result(
         disable_geoip=None,
     ) -> Optional[FeatureFlagResult]:
         """
-        Get a FeatureFlagResult object which contains the flag result and payload for a key by evaluating locally or remotely
-        depending on whether local evaluation is enabled and the flag can be locally evaluated.
-
-        This also captures the $feature_flag_called event unless send_feature_flag_events is False.
+        Get a FeatureFlagResult object
```

Comment:
This one isnt following your proposed structure

Reply:
Do you mean that this is different from other methods here or different form the JS doc comment structure.

If the latter,
This is a good point to discuss, this is closer to [google style python doc strings](https://google.github.io/styleguide/pyguide.html#doc-function-raises). I think a part of this was to keep this closer to:
1. what we already use (latest doc strings by olly follows closer to google stype doc strings)
2. what is native to this language.

I think it would look pretty wild to have ts/jsdoc in python code

Reply:
But I think you mean the former :P I think I missed a few methods 😆